drivers/gnss/gnss_nmea_generic.c

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright 2023 Google LLC
```

Comment:
should have prob left the original copyright since this is very the same quectel codebase